src/overview/reducer.js

```diff
@@ -5,13 +5,16 @@ import * as actions from './actions'
 const defaultState = {
     searchResult: {rows: []},
     query: '',
+    isFetching: true,
 }
 
 function setQuery(state, {query}) {
+	state.isFetching = true; //will start fetching
```

Comment:
Actaully I think instead of will start fetching. I should elaborate a little while intializing default